globus_automate_client/cli/actions.py

```diff
@@ -65,7 +65,8 @@ def action_run(
         callback=json_validator_callback,
     ),
     request_id: str = typer.Option(
-        None, help=("An identifier to associate with this Action invocation request"),
+        None,
+        help=("An identifier to associate with this Action invocation request"),
```

Comment:
Not a complaint, but seems there's been some of this formatting stuff bouncing back and forth. I wonder if we somehow have different settings on black or if it is not consistent itself or whatever.

Perhaps something to be considered if we see this happening more. Definitely outside the scope of this PR.

Reply:
I noticed that too...  I've felt that VS code has been on the fritz lately and it might not be formatting correctly on my end. Will keep an eye out for when it happens again